src/content/en/2020/fonts.md

```diff
@@ -22,3 +22,266 @@ queries: 04_Fonts
+
+The COLR vector format is supported on all major modern browsers, but only fairly recently. The fourth format is embedding SVG in OpenType (not to be confused with SVG fonts), but not supported in Chrome or Firefox, largely because SVG is a heavyweight format and difficult to render efficiently.
+
+Generally the bitmap format has larger files and doesn’t scale as cleanly to larger sizes, but vectors only offer flat color shading (though work is ongoing to offer richer shading options in a future version).
+
+Probably most usage of color fonts is for emoji, but the capability is general purpose and color fonts offer many design possibilities. While color web fonts haven’t taken off yet, the underlying technology is heavily used to deliver system emoji, where file format compatibility is much less of an issue.
+
+Browser support is so fragmented that color fonts are not yet tracked by caniuse.com, though there is [an issue open for it](https://github.com/Fyrd/caniuse/issues/1018).
+
+Lots more information about color fonts, including examples, are available at [colorfonts.wtf](https://www.colorfonts.wtf/).
+
+## Variable fonts
+
+{{ figure_markup(
+  image="fonts-font-variation-settings-usage.png",
+  caption="Usage of font-variation-settings axes.",
+  description="A bar chart showing the usage of the 6 most popular axes for font variation settings.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT2Q4hcDGGdclJH2ym0Pp_f8JWvYur_OQFQNkuScJyO7_ZCR1KPZsewL-mEZhxcuRFcde_Mxio8z_8P/pubchart?oid=173491049&format=interactive",
+  sheets_gid="655527383",
+  sql_file="04_17.VF_axis_value.sql"
+  )
+}}
+
+Variable fonts are certainly one of the biggest stories this year. They’re seen in 8.37% of desktop pages, and 13.84% of mobile. That’s up from an average of 1.8% last year, a huge growth factor. It’s not hard to see why their popularity is increasing - they offer more design flexibility, and also potentially smaller binary font sizes, especially if multiple styles of the same font are used on the same page.
+
+By far the most commonly used axis is `wght` (which controls weight), at 84.7% desktop and 90.4% mobile. However, `wdth` (width) accounts for approximately 5% of variable font usage. In 2020, Google Fonts began serving 2-axis fonts with both width and weight axes.
+
+The optical size (`opsz`) feature is used for approximately 2% of the variable font usage. This is one to watch, as tuning the appearance of a font to match its intended size of presentation improves the visual refinement in perhaps subtle but very real ways.
```

Comment:
Perhaps add "Usage is also likely to increase once some current cross-browser and cross-platform uncertainties on how the optical sizes are defined are cleared up."